docs/docs/aleph_zero/sdk.md

```diff
@@ -85,8 +99,10 @@ const account = keyring.addFromUri('//Alice')
 
 ### Initialize DEX and tokens
 
+Load the Invariant contract by providing the Polkadot API (`api`), specifying the network (e.g., `Network.Local` for local development), and indicating the Invariant contract address (`INVARIANT_ADDRESS`). Similarly, initialize the PSP22 token contract using the same approach. It's noteworthy that only a single instance of PSP22 is required to handle interactions with multiple tokens.
+
 ```typescript
-// --snip--
+// -- snip --
```

Comment:
I am skeptical about adding `// -- snip --` at the beginning of every code snippet. Everything mentioned is included within its own context